src/backend/dx12/src/device.rs

```diff
@@ -751,25 +762,31 @@ impl d::Device<B> for Device {
 
         let module = spirv::Module::new(spirv_data);
 
-        let parse_options = spirv::ParserOptions::new();
-        let ast = spirv::Parser::new(CompileTarget::Hlsl)
+        let parse_options = spirv::ParserOptions::default();
+        let parsed = spirv::Parser::new()
             .parse(&module, &parse_options)
             .map_err(|_| d::ShaderError::CompilationFailed("Unknown parsing error".into()))?;
```

Comment:
FYI `spirv_cross` [tries to retrieve an error message now in `ErrorCode::CompilationError(String)`](https://github.com/grovesNL/spirv_cross/blob/master/spirv_cross/src/lib.rs#L26) - could be useful for some initial debugging if we update the quad shader.